languages/go/exercises/concept/basic-slices/.docs/hints.md

```diff
@@ -0,0 +1,26 @@
+# Hints
+
+### General
+
+- slices in Go are zero-based. The first index in a slice is `0`.
+
+### 1. Retrieve a card from a stack
+
+- To get the `n`th item of a slice [use an index](https://blog.golang.org/go-slices-usage-and-internals).
+- To check if an item exists in a slice use a conditional and compare with `len(slice)`.
+
+Note: a slice always starts with index `0`, `1`, `2`, etc.
+
+### 2. Exchange a card in the stack
+
+- To set the `n`th item in a slice asign a new value to it: `slice[n] = x`.
```

Comment:
Maybe also use the same wording as above (without actually listing the syntax that is used)?

Reply:
Oh, yeah... should have checked for more places 😄

Reply:
:)

Reply:
@ErikSchierboom changed. Added some more links, too.